site/src/common/blocks/CallToActionListBlock.tsx

```diff
@@ -23,3 +34,7 @@ const Root = styled.div`
         gap: ${({ theme }) => theme.spacing.S400};
     }
 `;
+
+const GridColumn = styled.div`
+    grid-column: 3 / 23;
```

Comment:
I would prefer `3 / -3`. Makes it easier to see that it's centered / indented symmetrically. 
Also this would still work if `grid-template-columns` is changed to some other value. 

_(same for the other blocks)_

 ```suggestion
     grid-column: 3 / -3;
```